src/input_demo/main.cpy

```diff
@@ -18,7 +18,8 @@ class GestureWidget : public ui::Widget:
     // drag events
     gestures.drag_start += PLS_LAMBDA(auto & ev) {
       debug "DRAG START"
-      fill = GRAY
+      if ev.is_long_press:
+        fill = GRAY
```

Comment:
FYI for testing, the behavior now is that regular drag doesn't change the color of the rectangle (it just moves it), but long_press + drag changes the rectangle fill to GRAY.